packages/terra-compact-interactive-list/src/proptypes/cellShape.js

```diff
@@ -5,6 +5,11 @@ const cellShape = PropTypes.shape({
    * Content that will be rendered within the Cell.
    */
   content: PropTypes.node,
+
+  /**
+   * Boolean value indicating whether or not the cell should render as selected.
+   */
+  isSelected: PropTypes.bool,
```

Comment:
Shouldn't the cell shape have an isSelectable prop?  Wouldn't the cell be selected?

Reply:
Currently it is determined for the whole column rather than each cell and included in the column shape. The cell selection mechanism might be a subject to change in the future, current implementation might change in the future as it wasn't completely clear what is the best way for it.